src/style.css

```diff
@@ -0,0 +1,44 @@
+/* *{
```

Comment:
- [ ] Great job working on this milestone. Kindly remove all commented code from your codebase. It makes it look professional.

Reply:
Hello @t-yanick 👋🏾
Very nice to meet you, 
thank you for your suggestions,
 I appreciate it, I already modified the elements you suggested me.
 Best regards,